dev/idm-am-common-user/idm/conf/endpoint-staticuser.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This file is unnecessary in this sample. I only was using it for the privileged IG account I had defined for the proxy.